client/reader/search-stream/blank.jsx

```diff
@@ -1,5 +1,27 @@
-import React from 'react';
+const React = require( 'react' );
 
-export default function BlankContent() {
-	return <p />;
-}
+const BlankContent = React.createClass( {
+
+	propTypes: {
+		suggestions: React.PropTypes.array
+	},
+
+	shouldComponentUpdate: function() {
+		return false;
+	},
+
+	render: function() {
+		const suggest = this.props.suggestions
+			? <p>{this.translate( 'Staff Suggestions:')} { this.props.suggestions }.</p>
```

Comment:
would be good to use a component placeholder here to allow translators to move the set of suggestions around if necessary. something like `this.translate( 'Staff Suggestions: {{suggestions /}}', { components: { suggestions: ... } } )`